src/code/Microsoft.PowerShell.TextUtility.csproj

```diff
@@ -8,7 +8,7 @@
         <AssemblyVersion>1.0.0.0</AssemblyVersion>
         <FileVersion>1.0.0</FileVersion>
         <InformationalVersion>1.0.0</InformationalVersion>
-        <TargetFramework>netstandard2.0</TargetFramework>
+        <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
Are you using something from 2.1?  .NET Framework I think only supports up to 2.0 so this wouldn't work with WinPS

Reply:
yah - the `Array.Fill` does not exist in netstandard2.0

Reply:
However, I can do the same thing with a little additional code.